### PR TITLE
Balance: Buff Biochemical Cells

### PR DESCRIPTION
**Pull request by warp-core**
**Balance?**

## Summary
Significantly increase the energy capacities of the small and large Biochemical Cells.
3k -> 7.5k on the small.
7k -> 30k on the large.
Also, reduce the price of the nGVF-AA Fuel Cell from 40k to 35k because it seemed a little too close to the nGVF-BB Fuel Cell and raising the price on that would put it too close to the cell above and just lead to inflation of everything.

~~25% increase in energy and heat generation for the two Biochemical Cells, and increase the energy capacity of the large Biochemical Cell so it has slightly higher energy capacity/ton than the small, instead of significantly lower as it was before.
Buffs were chosen by @Amazinite in response to my suggestion that the proposed Hailstone fighters would need Red Sun Reactors because Small Biochemical Cells don't really produce a lot of power.~~

### CODE REVIEW

**Comment by Zitchas** [comment]
Could you provide a table or some data on comparisons? I personally think the Korath generators should be more than a "little" better in comparison (once cooling and storage are taken into account)

But that aside, buffing them a bit sounds good.

How does this improve them compared to Human and Hai stuff?

**Comment by warp-core** [comment]
generator | cost | size | energy/sec | heat/sec | capacity | energy/ton | energy/heat
-- | -- | -- | -- | -- | -- | -- | --
Small Biochem | 230000 | 17 | 96 | 84 | 3000 | 5.65 | 1.14
^ buffed | 287500 | 17 | 120 | 105 | 3000 | 7.06 | 1.14
Large Biochem | 1175000 | 63 | 372 | 294 | 7000 | 5.90 | 1.27
^ buffed | 940000 | 63 | 465 | 367.5 | 12000 | 7.38 | 1.27
Red Sun | 3800000 | 47 | 498 | 780 | 2200 | 10.60 | 0.64
Yellow Sun | 7500000 | 82 | 912 | 1260 | 5200 | 11.12 | 0.72
Candle | 493000 | 23 | 186 | 420 |   | 8.09 | 0.44
Furnace | 925000 | 39 | 330 | 780 |   | 8.46 | 0.42
Inferno | 1877000 | 71 | 624 | 1620 |   | 8.79 | 0.39
Plasma Core | 4188000 | 54 | 714 | 2160 |   | 13.22 | 0.33
DPC | 9321000 | 93 | 1320 | 4320 |   | 14.19 | 0.31
AA Fuel Cell | 40000 | 20 | 72 | 114 |   | 3.60 | 0.63
BB Fuel Cell | 47000 | 30 | 114 | 174 |   | 3.80 | 0.66
CC Fuel Cell | 72000 | 40 | 156 | 228 |   | 3.90 | 0.68
DD Fuel Cell | 118000 | 60 | 246 | 342 |   | 4.10 | 0.72
EE Fuel Cell | 177000 | 80 | 354 | 468 |   | 4.43 | 0.76
RT-I | 300000 | 45 | 264 | 1080 |   | 5.87 | 0.24
S3 | 360000 | 55 | 300 | 720 |   | 5.45 | 0.42
NT-200 | 570000 | 65 | 432 | 900 |   | 6.65 | 0.48
Dwarf Core | 800000 | 36 | 288 | 840 |   | 8.00 | 0.34
Fission | 1500000 | 65 | 564 | 1500 |   | 8.68 | 0.38
Breeder | 2500000 | 80 | 768 | 2100 |   | 9.60 | 0.37
Sand | 45000 | 24 | 99 | 147 |   | 4.13 | 0.67
Pebble | 1050000 | 32 | 270 | 714 |   | 8.44 | 0.38
Geode | 1750000 | 58 | 534 | 1272 |   | 9.21 | 0.42
Boulder | 5750000 | 90 | 1038 | 2040 |   | 11.53 | 0.51
Small Cogen | 176000 | 11 | 69 | 51 |   | 6.27 | 1.35
Large Cogen | 799000 | 46 | 318 | 207 |   | 6.91 | 1.54
Small Reactor | 5497000 | 42 | 588 | 1140 |   | 14.00 | 0.52
Large Reactor | 17754000 | 98 | 1452 | 2700 |   | 14.82 | 0.54


</body>

</html>

**Comment by warp-core** [comment]
> I personally think the Korath generators should be more than a "little" better in comparison (once cooling and storage are taken into account)

Any particular reason?

**Comment by Zitchas** [comment]
Well, our lore is basically that the Korath are experts in terms of "Produce the most energy at all costs." Said cost typically being a lot of heat. In contrast, the Wanderers are, to quote the Biochemical reactors:

> "The Wanderers are masters of unusual and efficient technology. This bioreactor is fueled by a culture of microorganisms that produce a small electric charge.

To me, that sounds like a reactor that doesn't necessarily produce a lot of energy per space, but also has virtually zero heat produced. Maybe even is an endothermic reaction (like their thrusters). So great power sources for fighters, interceptors, or just avoiding needing to deal with heat at all; but not a great choice when high energy production is required.

**Comment by Zitchas** [comment]
Also worth noting that we've got two use-cases in terms of balance:
1. The meta-game; which is what we tend to focus on as developer community; which focuses on extremes and max-efficacy builds pulling from every available faction ecosystem.
2. The new player who doesn't either doesn't realize that capturing and looting are options; or don't know what else is out there to use.

While I really appreciate all the work we do balancing for the first group; we need to ensure that the ecosystems of outfits are internally consistent and sufficient for people in the second group.

Flying super-optimized ships can be fun, but personally I find flying and outfitting ships entirely within the scope of their own ecosystem is both more challenging and more rewarding. :)

**Comment by warp-core** [comment]
Leaving the energy generation as is (or reducing it), and instead increasing the heat efficiency would serve little to no purpose on any Wanderer ships that use these generators.

The Earth Shaper (SBC) already has plenty of energy and runs with negative idle heat.

The Deep River (LBC) cannot quite power its engines, weapons, and shield generator all at once but has negative idle heat. With this buff, the idle heat is still negative and its energy generation is slightly higher than its consumption at full power.

The Deep River Transport has a Yellow Sun, which still can't quite power all of its systems at once. It lacks the space to install 2 LBCs (which, after this buff, would just barely exceed the energy generation of a Yellow Sun) and, even if it could, any reductions in heat would not be useful as it is far from overheating without a lot of incoming heat damage.

A Cool Breeze (Red Sun) will run out of power it ~11 seconds at full power. It will not overheat on its own. With 3 SBCs (it has 11 free outfit space to begin with, these will leave 7 free) it runs even cooler and actually takes longer to run out of power (22 seconds before the buff, 27 after) because of the much higher energy capacity. It actually already makes more sense to replace the Red Sun with a trio of SBCs.

The Summer Leaf (Yellow Sun) has a significant amount of free outfit space and so can install 2 LBCs.
With the Yellow Sun, it is close to its thermal limit but will not exceed it on its own. It will run out of power in just over 54 seconds.
With 2 LBCs, it is much cooler, notably heavier and less manoeuvrable, and will run out of power in 52 seconds (pre buff), but much longer after the buff. Ignoring the mass increase, against the Hai, the unbuffed LBCs make little sense, but could become relevant against Kor Sestor or Exile weapons which apply heat damage. After the buff, the mass increase is still present, but the increased energy capacity and thermal overhead are definitely things to consider.
It can also fit a White Sun instead of the Yellow Sun. With this configuration, it has a large excess in power generation, and a not insignificant store, perfect for fighting the Hai. It will overheat at full power, however. Though, since its normal energy generation exceeds its maximum energy consumption, its idle heat will normally be lower and so it will not overheat unless it is affected by heat, burn, ion, or energy damage.

The Autumn Leaf starts with a single White Sun. This gives it a slight energy excess, plus the storage, again, good against the Hai. Its total heat slightly exceeds its max heat, but the difference is small enough that it takes a long time of continuous firing, turning, and thrusting to get there. 2 LBCs are an option, but not a great one. It does run much cooler, negative idle heat even after this buff, but it no longer has an energy generation excess and will run out of power in 30 seconds. After the buff, it can last much longer, and still maintains low idle heat. This is actually a reasonable configuration against high heat damage opponents, though the ship is too weak to survive against a default Met Par Tek 53 to overheat under any of these configurations before it is disabled, though it gets close with the White Sun.

The Winter Gale starts with a White Sun, runs out of power in 18 seconds, doesn't overheat on its own.
With 2 LBCs, it'll run out of power in under 10 seconds (before the buff) but has significantly more thermal headroom.
With the White Sun, against a normal MPT53, it doesn't last long enough to run out of power or overheat, but is closer to overheating.
With 2 LBCs, without this buff, it runs out of power but is not close to overheating. The MPT53 wins more withmore health remaining. Definitely a worse option than the White Sun.
After the buff, it doesn't run out of power or overheat, I think closer to running out of power. A reasonable alternative to the White Sun.

A Strong Wind starts with a White Sun, will eventually run out of power and overheat, but overheating takes much longer. Not a good choice to use 2 LBCs here against the Hai. Against 2 MPT53s, both configurations lose, the White Sun is about to overheat, the LBCs are about to run out of power. Buffing the LBCs means more power is left but it still can't win.

The Tempest uses a pair of Yellow Suns. The Biochemical alternative is 2 large and 2 small. Against two MPT53s:
With the 2 Yellow Suns, it overheats with about half its hull left.
With the unbuffed biochemical cells, it almost runs out of power just before it is disabled.
With the buffed biochemical cells, it neither runs out of power or overheat, but still loses.

The Derecho uses a Blue Sun and a Red Sun and has a Heat Sink. It'll run out of power in just under 30 seconds, and won't overheat on its own. It could instead take 3 large and 1 small biochemical cells (uninstalling the heat sink does not give enough additional space for another biochemical cell).
Unbuffed, this will run out of power in under 20 seconds, but has much more thermal overhead.
Buffed, it still has plenty of thermal overhead but runs out of power in 22 seconds.
Against two MPT53s, it is close to overheating (83%) but lasts long enough to win.
Against three, it is defeated before overheating.
With the biochemical cells (unbuffed) against 2, it runs out of power and the second MPT53 lasts long enough to win. Against three, it runs out of power and is then disabled.
With the buffed biochemical cells, against 2, it is close to running out of power, but is able to defeat both enemies first. Against 3, it is again defeated before overheating.

The Hurricane. With its Blue and White Suns, it'll run out of power in 24 seconds, and overheat at some point.
With 4 large and 2 small Biochemical Cells (unbuffed), it'll run out of power in 11 seconds, but never overheat on its own.
After the buff, it'll run out of power in 21 seconds, and still never overheat on its own.
Against 3 MPT53s, the default config overheats with about half its hull left.
The unbuffed biochemical config, it runs out of power with about 3/4 of its hull left.
With the buffed biochemicals, it is close to running out of power but is disabled first.

The Flycatcher has 2 Small Biochemical Cells. Before the buff, it'll eventually run out of power, but not overheat on its own. After the buff, it'll do neither on its own.

Overall, lowering the heat output, even making it negative, will do little to give these generators a place in the Wanderer ecosystem. This buff widens the set of situations where they might be useful on Wanderer ships and will make it a little more reasonable to power a Hailstone's 2 Moonbeams while keeping its outfit space down.

**Comment by Amazinite** [comment]
If we're leaving the energy generation on the biochemicals the same, the only real alternative I see is not to reduce their heat generation, but to increase their energy capacity. With the values from this PR, the small biochemical has an energy density of about 176/ton, while the large has an energy density of about 190/ton. By comparison, the sun reactors have energy densities ranging from about 47 to about 117, but dedicated battery outfits have energy densities of at least 400. So what we could do is flip the purpose of the biochemicals; instead of being power generation that has a small amount of energy capacity added on, have them be good batteries that also slowly regenerate themselves.

**Comment by Zitchas** [comment]
I like that idea. Having them not be power generation (the Wanderers already have plenty of those and get more), but instead be batteries with the novel perk of filling themselves up slowly over time sounds like a new and interesting addition to the energy supply "field." Much, much more interesting than just yet another generator with slightly different combinations of generation, heat, and space.

**Comment by warp-core** [comment]
Sounds good to me. While we're here, I think the nGVF-AA Fuel Cell is too expensive. I think 35000 would be more ine line with the others in its series and the Sand Cell.

**Comment by Amazinite** [comment]
The price increase seems unnecessary.

**Comment by ravenshining** [comment]
Not that I particularly care, but I question whether things like biochemical cells and solar panels need to be good or competitive, stat-wise.  I've always assumed that they exist and are used for sustainability or environmental reasons, that their lower stats represent lower external costs, and the reason they are used is not because they are economical or efficient, but because they are *moral*.

If we really wanted to, we could even model this - have conventional sources of power flag a ship in such a way that disallows certain missions involving premium customers or radiation-sensitive passengers or cargo, similar to luxury accommodations or brigs.

**Comment by Zitchas** [comment]
Ravenshining raises a very good point, and I feel in some respects it parallels my early comment: There are a  *lot* of reasons to use a particular outfit outside of its suitability for combat min/maxing (which typically focuses most heavily on a "per-space" density). Just because we, as players, don't value or appreciate those reasons doesn't make them any less real.

Alternate reasons for something being used:
- Cheaper (and stuff is usually cheaper because of one or more of the following)
- Easier to manufacture
- Doesn't require rare resources (or at least significantly less than other things of similar type)
- Safer (to manufacture, operate and/or dispose of)
- Environmental impacts (of manufacture and/or operation)
- Sustainability (X can be made of recycled components when the higher performance Y requires all-new components)
- Requires less training to operate and maintain.
- Noise (there are a *lot* of use cases in the world where an otherwise poor piece of equipment is used instead of better things merely because it is quieter. More than a few people under-clock their computers, for example, by significant margins in order to get a system that can run 100% on passive cooling with no electronics whine or fan noises)
- Marketing. (our economy is *filled* with garbage that is worse than the alternatives by *every* metric including price, but is still produced, sold, and used simply because it has a good marketing team promoting it, and/or a big advertising budget)
- Feel-good factor: (This item is made by *insert marginalized group here* and/or supports *insert feel-good cause here*.)


Long story short, there's zero reason that things *must* be competitive in performance per space/heat/energy. But they do need story/lore as to why they're in-use. Building up lore and story is better for our universe than ensuring everything is competitive for space-combat. We have to remember that our ships and outfitting have to support a *much* broader range of use cases than just combat, even though that's pretty much our only major metric for *us* using them.

**Comment by Amazinite** [comment]
Outfits can both exist for lore purposes while also being useful to the player in some way, shape, or form. I had this conversation with Zitchas about energy blasters; energy blasters can, and I think now do, have a proper place in the weapon sandbox without being completely useless. They can have a lore reason for existing as the worst weapon in human space while still being a viable choice as the worst weapon. I do not think that we should have outfits whose lore is literally, "This outfit is absolutely terrible to use for any reason and anyone in their right mind would replace it right away, but it's kept around for XYZ reasons."
